### ISSUE TITLE
Could the simple mode output rtab file?

### ISSUE THREAD

**Issue by Tonny-zhou** [johnlees/unitig-caller#17]
Dear author,

Thank you very much for this great tools! I have a small request...could the simple mode output the rtab-format file rather than the pyseer format file?

**Comment by johnlees** [comment]
Yes this would be possible, I might do this if there are more requests for this feature

I don't expect to have time in the near future so would suggest converting with a script for now

**Comment by MetteBoge** [comment]
I second that!
Do you have a script for converting a pyseer to rtab?

**Comment by johnlees** [comment]
Thanks for the interest – I am not going to add any new features to this package due to lack of capacity. But if anyone interested were to make a script or PR (with LLM input certainly acceptable) I would be happy to review and merge it
